tools/testsys/src/aws_resources.rs

```diff
@@ -43,30 +43,57 @@ impl AwsK8s {
         test: TestType,
         testsys_images: &TestsysImages,
     ) -> Result<Vec<Crd>> {
-        match test {
-            TestType::Conformance => {
-                self.sonobuoy_test_crds(testsys_images, SonobuoyMode::CertifiedConformance)
-            }
-            TestType::Quick => self.sonobuoy_test_crds(testsys_images, SonobuoyMode::Quick),
-            TestType::Migration => self.migration_test_crds(testsys_images).await,
+        let mut crds = Vec::new();
+        let target_cluster_names = if self.config.cluster_names.is_empty() {
+            debug!("No cluster names were provided using default name");
+            vec![self.default_cluster_name()]
+        } else {
+            self.config.cluster_names.clone()
+        };
+        for template_cluster_name in target_cluster_names {
+            let cluster_name = &rendered_cluster_name(
+                template_cluster_name,
+                self.kube_arch(),
+                self.kube_variant(),
+            )?;
+            crds.append(&mut match &test {
+                TestType::Conformance => self.sonobuoy_test_crds(
+                    testsys_images,
+                    SonobuoyMode::CertifiedConformance,
+                    cluster_name,
+                )?,
+                TestType::Quick => {
+                    self.sonobuoy_test_crds(testsys_images, SonobuoyMode::Quick, cluster_name)?
+                }
```

Comment:
Seeing this makes me wonder how expanding this list would look like. For example, if I want to run a different sonobuoy test mode like `conformance-lite`, do I add to the `TestType` enum? But `TestType` enum needs to be generic across all tests including ECS testing where `conformance-lite` doesn't make sense.

Reply:
We will add the `conformance-lite` test type and then report that it is unavailable for the other variants if a user tries to use them.

Reply:
Isn't the `conformance-lite` test type in sonobuoy just a NGINX pod? If so, we could have the same `conformance-lite` testing for ECS and support both orchestrators.

Reply:
You're thinking of the quick test mode. `conformance-lite` is a subset of tests from the certified K8s conformance tests suite that is faster to run.